projboard/tests/test_like.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from projboard.models import Like, User, Article, Subject
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+NAME = "Shalev"
+
+
+@pytest.mark.django_db
+class TestLikeModel:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_subjects(self):
+        sup = Subject(name='N')
+        sup1 = Subject(name='R')
+        sup.save()
+        sup1.save()
+        return [sup, sup1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_user(self):
+        user = User(name=NAME, nickname="S99", email="Shalev@gmail.com", password="Shalev123")
+        user1 = User(name="John", nickname="J99", email="John@gmail.com", password="John123")
+        user.save()
+        user1.save()
+        return [user, user1]
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_articles(self, create_user, create_subjects):
+        articles_list = []
+        for i in range(len(create_user)):
+            article = Article(user_id=create_user[i], title=TITLE,
+                              content=CONTENT, subject_id=create_subjects[i])
+            article.save()
+            articles_list.append(article)
+
+        return articles_list
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_like(self, create_user, create_articles):
+        like = Like(user_id=create_user[0], article_id=create_articles[0])
+        like.save()
+        return like
+
+    def test_create_like(self, create_like):
+        assert create_like.user_id.name == NAME
+        assert create_like.article_id.title == TITLE
+
+    def test_delete_like(self, create_user, create_like):
+        result = Like.delete_like(create_user[1], create_like.article_id)
```

Comment:
Please see above
avoid usage [0] or [1] -> use [i] or a class based on fixtures calls

Reply:
Hi @Yarboa,
Please take a look at [033bfc9](https://github.com/redhat-beyond/Smarticle/pull/30/commits/033bfc9d62066b95448471aa9a9d1797d788b9ec).
I Changed the fixtures and a some tests